firmware/integration/rusefi_config.txt

```diff
@@ -93,7 +93,7 @@
 ! Any time an incompatible change is made to the configuration format stored in flash,
 ! update this string to the current date! It is required to also update TS_SIGNATURE above
 ! when this happens.
-#define FLASH_DATA_VERSION 10019
+#define FLASH_DATA_VERSION 10020
```

Comment:
@Er0pification that's why FLASH_DATA_VERSION change to force configuration migration via TS